service/src/main/java/uk/gov/hmcts/reform/fpl/model/event/UploadDraftOrdersData.java

```diff
@@ -159,4 +161,17 @@ public Object getHearingDynamicList() {
 
         return null;
     }
+
+    @JsonIgnore
+    public YesNo hasDraftOrderBeenUploadedThatNeedsApproval() {
+        // if we've got a C21 draft order - these are always approved
+        if (isNotEmpty(hearingOrderDraftKind) && hearingOrderDraftKind.contains(C21)) {
+            return YesNo.YES;
+        }
+        // if we've got a CMO draft order - these are only approved if they're agreed CMOs
+        if (isNotEmpty(cmoUploadType) && cmoUploadType.equals(AGREED)) {
```

Comment:
I imagine it will be simpler - `AGREED.equals(cmoUploadType)`

Reply:
ah yes agreed! technically NPE safer too 😄